pkg/minishift/addon/parser/addon_parser_test.go

```diff
@@ -51,6 +52,15 @@ var noDescription string = `# Name: One command of each type
 oc foo
 `
 
+var addOnWithMultilineDescription string = `# Name: foo
+# Description: This is one line
+#   This is second line
```

Comment:
Maybe we can add some more tests as well. Some tests with other meta tags following the multi lines description and if we decide to use indentation, I would like to see a test for something like:

    # Name: foo
    # Description: line one
    #   line two
    # No indentation - what happens with this line
    #  Required-Vars: ...